src/utils/destinantionCallParams.ts

```diff
@@ -0,0 +1,15 @@
+import { DestinationCallDataParams } from '@connext/chain-abstraction/dist/types';
```

Comment:
lets create a connext.ts file inside utils to send all of these new utils

Reply:
Works